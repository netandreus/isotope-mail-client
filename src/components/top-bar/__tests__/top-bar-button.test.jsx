import React from 'react';
import {shallow} from 'enzyme/build/index';
import TopBarButton from '../top-bar-button';

describe('TopBarButton component test suite', () => {
  test('Snapshot render, should render top-bar-button', () => {
    // Given
    const props = {onClick: jest.fn()};
    // When
    const button = shallow(<TopBarButton {...props}>icon_name</TopBarButton>);
    // Then
    expect(button).toMatchSnapshot();
  });
});
