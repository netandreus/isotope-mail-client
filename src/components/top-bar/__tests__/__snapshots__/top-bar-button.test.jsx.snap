// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBarButton component test suite Snapshot render, should render top-bar-button 1`] = `
<IconButton
  className=" mdc-top-app-bar__action-item"
  disabled={false}
  onClick={[MockFunction]}
>
  icon_name
</IconButton>
`;
